packages/components/src/Well/Well.module.scss

```diff
@@ -12,7 +12,55 @@
   margin: 0;
 }
 
-// Background Style Variations
+.white {
+  background-color: $color-white;
+  border-color: $color-gray-500;
+  color: $color-purple-800;
+}
+
+.gray {
+  background-color: $color-gray-200;
+  border-color: $color-gray-500;
+  color: $color-purple-800;
+}
+
+.blue {
+  background-color: $color-blue-100;
+  border-color: $color-blue-400;
+  color: $color-purple-800;
+}
+
+.yellow {
+  background-color: $color-yellow-100;
+  border-color: $color-yellow-700;
+  color: $color-purple-800;
+}
+
+.orange {
```

Comment:
No need for orange, `assertive` should be deprecated :)

Reply:
Ah true, added that in and forgot to remove it